.github/workflows/ci-tests.yml

```diff
@@ -227,9 +227,10 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: '3.10'
+    - uses: actions/checkout@v2
     - name: Install dependencies
       run: |
-        pip install git+https://github.com/${{ github.event.pull_request.head.repo.full_name }}@${{ github.head_ref }}
+        pip install .
```

Comment:
Probably better, since it's more self contained.

One nitpicky, random question: Should we consider using `pip install -e .`? I'm not saying we _should_, but when I'm installing from source, that's the typical way to do it (mainly to allow edits). 🤷‍♂️

Reply:
So this test is specifically to make sure the program still works when installed like a user would via pip.

editable installing basically just creates a symlink(-like) to the code, whereas non-editable copies the package to the environment's `site-packages`. It's best to do non-editable for an integration test like this as it would also catch things like missing necessary package data from setup.cfg.